src/app/app-routing.module.ts

```diff
@@ -6,7 +6,7 @@ import { LogoutComponent } from './auth/logout/logout.component';
 
 
 const routes: Routes = [
-  { path: '', redirectTo: 'example', pathMatch: 'full' },
+  { path: '', redirectTo: 'projects', pathMatch: 'full' },
   { path: 'login', component: LoginComponent },
   { path: 'logout', component: LogoutComponent, canActivate: [ AuthGuard ] },
   { path: 'example', canActivate: [ AuthGuard ], loadChildren: './example-module/example-module.module#ExampleModuleModule' },
```

Comment:
redirectTo powoduje, że zmienia się adres na taki, jaki jest podany. Tutaj nie ma ścieżki projects (bo zostało example 3 linijki niżej, a sama zmiana tego example nie pomoże, bo powinno to przekierowywać pewnie do projectsModule)

Reply:
jest ścieżka `projects` - 4 linijki niżej :P

Reply:
a faktycznie